roles/node_file_backup/README.md

```diff
@@ -0,0 +1,57 @@
+# ibm.storage_protect.backup
+
+## Description
+An ansible role to take on demand backup on Storage protect client.
+
+## Requirements 
+- Ansible 2.9+ 
+- IBM Storage Protect server configured and accessible  and atleast one BA client node registered with server.
+
+## Environment Variables
+
+Before running the playbook, set the following environment variables in your terminal:
+
+```bash
+export STORAGE_PROTECT_SERVERNAME="<Storage protect server name that client is registered with>"
+export STORAGE_PROTECT_NODENAME="<Node name registered with storage protect server>"
+export STORAGE_PROTECT_NODE_PASSWORD="<Node password of the node that is registered with storage protect server"
+```
+
+## Variables
+
```

Comment:
You should now mention that these variable are the child of `storage_protect_node_file_backup`.

Once this is fixed then it should be good